Task cinema seats booking

ik-web opened the pull request:
##HTML movie seat booking

[Demo](https://ik-web.github.io/html-movie-seat-booking/)

Please, review!

ik-web committed:
Task cinema seats booking

ik-web committed:
Fixed index.html

Changed tag header to head
Added attribute lang to html
Added open tag li to first row of cinema hall

A-Ostrovnyy commented on submissions/ik-web/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,280 @@
+
+                <ul class="movie-info__about">
+                    <li class="movie-info__about-item about-item__format">3D</li>
+                    <li class="movie-info__about-item about-item__map">Kyiv, SEC "Lavina", Hall 13</li>
```
For address use HTML tag `addres` [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)

ik-web replied:
I fixed this point, thanks for the comment!

A-Ostrovnyy commented on submissions/ik-web/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,280 @@
+                <ul class="movie-info__about">
+                    <li class="movie-info__about-item about-item__format">3D</li>
+                    <li class="movie-info__about-item about-item__map">Kyiv, SEC "Lavina", Hall 13</li>
+                    <li class="movie-info__about-item about-item__date">2021.02.04 Thursday</li>
```
For time/date use HTML tag `time`  [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

ik-web replied:
I fixed this moment

A-Ostrovnyy commented on submissions/ik-web/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,280 @@
+                    <ol class="cinema-hall__row">
+                        <li>
+                            <label>
+                                <input type="checkbox" class="cinema-hall__check" value="1">
```
Add styles on `:hover` and `:focus` states.

ik-web replied:
Added. Thank you for this comment!

A-Ostrovnyy reviewed (changes requested):
Good work!
See my comments below.)

ik-web committed:
Fixed some mistakes

Added tags address and time to movie-info
Added pseudo-class :hover to cinema-hall seats input

ik-web committed:
Merge branch 'main' into ik-web/html-movie-seat-booking

A-Ostrovnyy reviewed (approved):
Well done!
